crates/legacy/src/service.rs

```diff
@@ -61,8 +61,8 @@ use std::{fmt::Display, time::Instant};
 use tagged_base64::TaggedBase64;
 use tide_disco::{method::ReadState, Url};
 
-// Start assuming we're fine calculatig VID for 100 kilobyte blocks
-const INITIAL_MAX_BLOCK_SIZE: u64 = 100_000;
+// Start assuming we're fine calculatig VID for 5 megabyte blocks
+const INITIAL_MAX_BLOCK_SIZE: u64 = 5_000_000;
```

Comment:
Initially my reasoning for setting 100 kilobytes was that this is something we definitely will be able to handle on any reasonable hardware.
Bumping this to 5 megabytes will certainly lead to subsequent downscaling on some machines, but will prevent cases where we never start building blocks because all transactions we receive are more than our starting limit.